src/main/java/anab/Person/Person.java

```diff
@@ -0,0 +1,49 @@
+package anab.Person;
+
+public class Person {
+
+    String firstName, lastName, Age, Gender, EmailAddress;
```

Comment:
please add private access modifier for the declared variables, based on encapsulation principle.